code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -98,11 +110,205 @@
 		ticker.mode.update_necro_icons_added(Controller)
 		ticker.mode.update_all_necro_icons()
 		ticker.mode.risen.Add(Controller)
-	name += " ([rand(1,1000)])"
 
-/mob/living/simple_animal/hostile/necro/copy/ListTargets()
-	. = ..()
-	return . - creator
+	if(name == "zombie" || name == "skeleton")
+		name += " ([rand(1,1000)])"
+
+/mob/living/simple_animal/hostile/necro/zombie/Life()
+	..()
+	/*TODO
```

Comment:
Now, they'll happily eat on dead corpses in the vicinity, and wander, but they just don't care about potential targets and I'm not entirely sure why